### PR TITLE
Feature 12 – Email templates

### LINKED ISSUE #12: Email Templates
As a user, I would like to see Cube Cobra emails templated nicer.

When an email is generated, the email should have html and css to make it looks more professional.

### PR DESCRIPTION
**Pull request by zlitzsinger-bw**
Tested with email previews and with Gmail. Fixes #12.

### CODE REVIEW

**Comment by dekkerglen** [comment]
Could you perform a rebase and recommit to fix the package lock?

File: emails/confirm_email/html.pug [hunk 1]
```diff
@@ -0,0 +1,12 @@
+link(rel='stylesheet' href='/css/email.css')
+
+div(class="banner-container")
```
Comment (crookedneighbor) [inline emails/confirm_email/html.pug]: This is the shorthand for the same markup. 

```suggestion
.banner-container
```

File: emails/confirm_email/html.pug [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    img(
+        src="https://cubecobra.com/content/banner.png" 
+        alt="Cube Cobra: a site for Magic: the Gathering Cubing" 
+        height="50px"
+    )
+
+p Thanks for joining Cube Cobra!
+
+a(class="button", href="https://cubecobra.com/user/register/confirm/"+id) Confirm my email
```
Comment (crookedneighbor) [inline emails/confirm_email/html.pug]:
```suggestion
a.button(href="https://cubecobra.com/user/register/confirm/"+id) Confirm my email
```

File: emails/password_reset/html.pug [hunk 3]
```diff
@@ -0,0 +1,16 @@
+link(rel='stylesheet' href='/css/email.css')
+
+div(class="banner-container")
```
Comment (crookedneighbor) [inline emails/password_reset/html.pug]:
```suggestion
.banner-container
```

File: emails/password_reset/html.pug [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    img(
+        src="https://cubecobra.com/content/banner.png" 
+        alt="Cube Cobra: a site for Magic: the Gathering Cubing" 
+        height="50px"
+    )
+
+p We received a request to reset your password. Your recovery code is: 
+code #{code}
+
+a(class="button", href="https://cubecobra.com/user/passwordreset/"+id) Reset password
```
Comment (crookedneighbor) [inline emails/password_reset/html.pug]:
```suggestion
a.button(href="https://cubecobra.com/user/passwordreset/"+id) Reset password
```

**Comment by dekkerglen** [comment]
I think I'm going to have to close this PR as the OP has not responded to resolving merge conflicts

**Comment by crookedneighbor** [comment]
If it’s an otherwise desired feature, I’m happy to pick it up.

**Comment by dekkerglen** [comment]
It's a desired feature, if you'd like to pick it up that would be excellent. Are you on the discord? If not, join here: https://discord.gg/Hn39bCU and @ me (dekkaru) to gain access to contributor channels. If you already are, I recommend changing your nickname on the Cube Cobra Discord server to mirror your github

**Comment by purxiz** [comment]
This should be closed probably, unless somebody wants to claim ownership

**Comment by crookedneighbor** [comment]
FYI, I had a branch with these changes brought up to date, but it never sent an email successfully. Just opened a new tab. Wasn’t sure if that was expected in dev mode. 

Also, the lack of tests made me real nervous to make any significant changes, so I decided not to pursue it further until I could get more familiar with the code base.

**Pull request closed without merging**